test/performance/broker-rmq/README.md

```diff
@@ -0,0 +1,78 @@
+- A git clone of [knative-sandbox/eventing-rabbitmq](https://github.com/knative-sandbox/eventing-rabbitmq) at tag v0.27.0 or higher
+We use [GCP `c2-standard-4`](https://cloud.google.com/compute/vm-instance-pricing#compute-optimized_machine_types)  machine type to make benchmarks results consistent. Using a different instance type, or a different cloud provider, is almost certain to yield different results.
+## 2/5. Install Knative Serving
+We install a specific version of Knative - [v0.26.0](https://github.com/knative/eventing/releases/tag/v0.26.0) - as this is a point-in-time guide. While we expect subsequent versions to continue working the same way, in the absence of automated tests that ensure this, we stick to exact versions that we have tested manually.
+```
+# Installing Knative Serving ...
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.26.0/serving-crds.yaml
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.26.0/serving-core.yaml
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.26.0/eventing-crds.yaml
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.26.0/eventing-core.yaml
+
+# Installing Knative Serving Kourier networking layer ...
+kubectl apply --filename https://github.com/knative/net-kourier/releases/download/v0.26.0/kourier.yaml
+# Patching Knative Serving to use Kourier for the networking layer ...
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+```
+> Aside: does this look right to you @benmoss ? https://github.com/knative/serving/releases/download/v0.26.1/serving-crds.yaml . I didn't install v0.26.1 - latest release - because of it.
+## 3/5. Install Knative Eventing RabbitMQ
+```
+# Installing RabbitMQ Operator ...
+kubectl apply --filename https://github.com/rabbitmq/cluster-operator/releases/download/v1.9.0/cluster-operator.yml
+# Installing cert-manager ...
+curl -sL https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml \
+| sed 's/kube-system/cert-manager/' \
+| kubectl apply --namespace cert-manager --filename -
+# Installing RabbitMQ Messaging Topology Operator with cert-manager integration ...
+kubectl apply --filename https://github.com/rabbitmq/messaging-topology-operator/releases/download/v1.2.1/messaging-topology-operator-with-certmanager.yaml
+# Installing Knative Eventing RabbitMQ Broker ...
+kubectl apply --filename https://github.com/knative-sandbox/eventing-rabbitmq/releases/download/v0.26.0/rabbitmq-broker.yaml
+```
+In case you want to test the latest dev version of eventing-rabbitmq, follow these steps:
+```
+# ⚠️ TODO after https://github.com/knative-sandbox/eventing-rabbitmq/pull/445 is merged:
+# git clone https://github.com/knative-sandbox/eventing-rabbitmq $GOPATH/src/knative.dev/eventing-rabbitmq
```

Comment:
Thanks for validating.
Would you mind filing a separate issue with your findings ?

Reply:
@benmoss helped me understand my issue: I was missing `$HOME/go/bin` from `PATH`.

It seems that when `GOPATH` is set, everything works as expected, since `GOBIN` defaults to `$GOPATH/bin`. I still think that we should keep this as is, and avoid explaining the non-GOPATH option.